src/ign.cc

```diff
@@ -153,3 +159,114 @@ extern "C" SDFORMAT_VISIBLE int cmdPrint(const char *_path)
 
   return 0;
 }
+
+//////////////////////////////////////////////////
+extern "C" SDFORMAT_VISIBLE int cmdInertialStats(
+    const char *_path)
+{
+  if (!sdf::filesystem::exists(_path))
+  {
+    std::cerr << "Error: File [" << _path << "] does not exist.\n";
+    return -1;
+  }
+
+  sdf::Root root;
+  sdf::Errors errors = root.Load(_path);
+  if (!errors.empty())
+  {
+    for (auto &error : errors)
+    {
+      std::cerr << "Error: " << error.Message() << std::endl;
+    }
+    return -1;
```

Comment:
I think we don't need to return here. There can be minor errors that wouldn't affect the inertial calculations. It's nice to print the errors, but just skip the `return` and that should be fine

Reply:
Wouldn't it be safer to return in case the error isn't minor? For example, it won't catch [inertial_invalid.sdf](https://github.com/ignitionrobotics/sdformat/blob/sdf9/test/sdf/inertial_invalid.sdf)

Reply:
Yeah that just broke a test case. ``ign sdf --inertial-stats`` should inherently check for sdf errors I think, and if those pass only then we should return the statistics. Maybe that is safer for users.

Reply:
it currently doesn't work for the VIPER urdf. That might be fixed by #959, but it's not very helpful for debugging if it doesn't at least try to give you some inertial information if there's anything at all wrong with the model. We could add a `--strict` flag to make this configurable if we can't agree, but I think printing the warning messages should be enough of a clue that there's something wrong

Reply:
Updated the test case. On the other hand maybe printing out the false inertial stats can help with debugging.